src/bp/ui-studio/src/web/components/Layout/StatusBar/TrainingStatus/MultiLang.tsx

```diff
@@ -0,0 +1,57 @@
+import { Icon, Popover, Position } from '@blueprintjs/core'
+import { NLU } from 'botpress/sdk'
+import { lang } from 'botpress/shared'
+import classNames from 'classnames'
+import React, { FC } from 'react'
+import { connect } from 'react-redux'
+import { RootReducer } from '~/reducers'
+import SingleLang from './SingleLang'
+import style from './style.scss'
+
+interface Props {
+  languages: string[]
+  trainSessions: { [lang: string]: NLU.TrainingSession }
+}
+
+const TrainingStatusCenter: FC<Props> = (props: Props) => {
+  const { languages, trainSessions } = props
+  return (
+    <div className={style.trainCenter}>
+      {languages.map(l => (
+        <div className={classNames(style.trainCenter_lang, style.trainStatus_message_dark)}>
+          <div className={style.trainCenter_lang_code}>{l}:</div>
+          <SingleLang dark={true} trainSession={trainSessions[l]} />
+        </div>
+      ))}
+    </div>
+  )
+}
+
+const MultiLangTrainingStatusComponent: FC<Props> = (props: Props) => {
+  const needsTraining = Object.values(props.trainSessions).some(ts => ts.status !== 'done')
+
+  return (
+    <React.Fragment>
+      {needsTraining && (
+        <Popover
+          content={<TrainingStatusCenter {...props} />}
+          minimal={false}
+          position={Position.TOP_RIGHT}
+          modifiers={{
+            arrow: { enabled: true },
+            preventOverflow: { enabled: true, boundariesElement: 'window' }
+          }}
+        >
+          <Icon className={style.trainCenter_icon} icon="predictive-analysis" />
+        </Popover>
+      )}
+      {!needsTraining && <span className={style.trainStatus_message_light}>{lang.tr('statusBar.ready')}</span>}
```

Comment:
I would also add a "training in progress when training is happening in any language"

Reply:
:+1: